cmd/workload/testsuite.go

```diff
@@ -98,14 +100,17 @@ func testConfigFromCLI(ctx *cli.Context) (cfg testConfig) {
 		cfg.fsys = builtinTestFiles
 		cfg.filterQueryFile = "queries/filter_queries_mainnet.json"
 		cfg.historyTestFile = "queries/history_mainnet.json"
+		cfg.mergeBlock = mainnetMergeBlockNum
 	case ctx.Bool(testSepoliaFlag.Name):
 		cfg.fsys = builtinTestFiles
 		cfg.filterQueryFile = "queries/filter_queries_sepolia.json"
 		cfg.historyTestFile = "queries/history_sepolia.json"
+		cfg.mergeBlock = sepoliaMergeBlockNum
 	default:
 		cfg.fsys = os.DirFS(".")
 		cfg.filterQueryFile = ctx.String(filterQueryFileFlag.Name)
 		cfg.historyTestFile = ctx.String(historyTestFileFlag.Name)
+		cfg.mergeBlock = math.MaxUint64 // placeholder until we implement the ability to plumb the merge block for custom networks thru the test config
```

Comment:
I don't think thats correct, since it means that all history is not available. I would also rename it to something else, like historyPruneBlock or something. We are not really interested in this being the merge block right? This is just because we want to put the prune block on the merge block

Reply:
Oof. yup you're right.

I have a pending commit where I replace it replace it with a `*uint64`, `nil` means that there isn't a "prune" block and the test will execute all of the cases.